server/app/services/applicant/ApplicantService.java

```diff
@@ -1101,4 +1113,128 @@ abstract static class Builder {
       abstract ApplicationPrograms build();
     }
   }
+
+  /** Get corrected address from Esri and formats it as a map compatible with form data */
```

Comment:
**suggestion (blocking):** please expand on what this method is doing and the contents of the return value

Reply:
done